tests/bofire/utils/test_torch_tools.py

```diff
@@ -875,6 +889,74 @@ def test_get_multiobjective_objective():
     assert np.allclose(objective_forward[..., 2].detach().numpy(), reward4)
 
 
+def test_get_additive_objective(mutiobjective_data):
```

Comment:
What is the difference between this test and `test_get_additive_botorch_objective` originally in line 287 of this file?